src/components/map/Map.tsx

```diff
@@ -129,63 +132,36 @@ const Map = ({
           style={{
             flex: 1,
           }}
-          onRegionDidChange={(region) => {
-            setRegionEvent({isMoving: false, region});
-          }}
-          onRegionWillChange={() =>
-            setRegionEvent({isMoving: true, region: regionEvent?.region})
-          }
-          onPress={flyToFeature}
+          onPress={onPress}
           {...MapViewConfig}
         >
           <MapboxGL.Camera
             ref={mapCameraRef}
-            zoomLevel={coordinates.zoomLevel}
+            zoomLevel={zoomLevel}
             centerCoordinate={[coordinates.longitude, coordinates.latitude]}
             {...MapCameraConfig}
           />
+          {mapLines && <MapRoute lines={mapLines}></MapRoute>}
```

Comment:
```suggestion
          {mapLines && <MapRoute lines={mapLines} />}
```
Empty body gives warning in my Intellij 😅